src/lib/api/api.common.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
??

Reply:
usage of any is not permitted , I am already using proper types in most places. but here  use of any is because data coming from backend can be of any type.  it will  be hard to narrow  it down to any proper interface atleast for now